switchboard/manager.py

```diff
@@ -22,6 +22,7 @@
 # any and all threads. The only exception to read-only is when they are
 # populated on Switchboard startup (i.e., operator.register()).
 registry = {}
+registry_by_namespace = {}
```

Comment:
What's the purpose of two registries? I'm guessing if there was only one registry, we'd have to re-architect other parts of the app to grok the new data structure, which is more than you wanted to tackle in this PR. If that's the case, let's get an issue created so we don't lose track of the work.